azure-devops/run-tests.yml

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+parameters:
+- name: buildOutputLocationVar
+  type: string
+  default: buildOutputLocation
+- name: targetPlatform
+  type: string
+- name: hostArch
+  type: string
+- name: targetArch
+  type: string
+- name: displayName
+  type: string
+  default: 'Run Tests'
+- name: runArtifact
+  type: boolean
+  default: false
+
+jobs:
+- job: 'test_${{ parameters.targetPlatform }}'
+  dependsOn:
+  - 'build_${{ parameters.targetPlatform }}'
+  condition: succeeded('build_${{ parameters.targetPlatform }}')

```

Comment:
Watch out for the job rename that Stephan just pushed to my PR branch. Also you're going to want to use the parallel strategy on this job like I did for the `Build ARM/ARM64` jobs.